app/src/main/java/org/p2p/wallet/home/ui/main/HomePresenter.kt

```diff
@@ -141,6 +141,20 @@ class HomePresenter(
     }
 
     override fun onSendClicked() {
+        view?.apply {
+            if (newSendEnabledFeatureToggle.isFeatureEnabled) {
+                val isAccountEmpty = state.tokens.all { it.isZero }
+                if (isAccountEmpty) {
+                    fallbackUsdcTokenForBuy?.let {
+                        view?.showSendNoTokens(it)
+                    }
```

Comment:
```suggestion
            fallbackUsdcTokenForBuy?.let { view?.showSendNoTokens(it) }
```

and why we cannot open buy without USDC?

Reply:
we just use default but yes i think we need to make sol also if there is no USDC